src/redux/reducers/character.js

```diff
@@ -8,12 +8,10 @@ import {
 const initialState = {
   name: '',
   moves: [], // all moves
-  moveFilter: [], // by frames, power, etc
   moveSearch: '', // search by name
```

Comment:
I would keep the filter here. ITs more confusing to have filters on the top level, if they are only used to effect the character moves.

Lets keep things with what they're used for